romana-install/roles/romana/install-agent/tasks/romana_from_github.yml

```diff
@@ -2,10 +2,33 @@
   command: rsync -az "{{ romana_master_ip }}:/var/tmp/gopath/bin/{{ item }}" "/var/tmp/{{ item }}"
   with_items:
     - agent
+    - romana
+
+- name: Copy policy-agent from master
+  command: rsync -az "{{ romana_master_ip }}:/var/tmp/gopath/src/github.com/romana/core/pkg/util/policy/agent/agent.py" "/var/tmp/agent.py"
```

Comment:
How does this get installed when using "binaries" instead of from source/github?